### PR TITLE
Basic component layout 1

### PR DESCRIPTION
**Pull request by moncapitaine**
ToDay work on our new startup

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,5 @@
 Harvest software dev hours
 
 Hello guys!
```
Comment (moncapitaine) [inline README.md:4]: Bad wording. Does not help

Comment (christianma24) [reply README.md:4]: Indeed

**Review by Wolfram2021 — COMMENTED** [review]
all clear

**Review by christianma24 — APPROVED** [review]

**Review by stewolf8 — APPROVED** [review]

File: frontend/README.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
-Unser Frontend
 
-create-react-app
\ No newline at end of file
+
```
Comment (equitable66) [inline frontend/README.md:2]: Gute Idee mit den Links.

**Review by ARKoellner — APPROVED** [review]
Delete .env and use .env.local instead

**Review by Christian1078 — COMMENTED** [review]
Alles Roger :-)

**Review by equitable66 — APPROVED** [review]

**Pull request merged**